tests/benchmark/utils/shared_info.cc

```diff
@@ -0,0 +1,105 @@
+#include "shared_info.h"
+
+extern "C" {
+#include "bin/common.h"
+
+}
+
+struct s2n_cipher_suite **all_suites = cipher_preferences_test_all_tls12.suites;
+unsigned int num_suites = cipher_preferences_test_all_tls12.count;
+
+
+int DEBUG_PRINT = 0;
+int WARMUP_ITERS = 1;
+unsigned int ITERATIONS = 50;
+int use_corked_io = 0;
+char bench_format[100] = "--benchmark_out_format=";
+char file_prefix[100];
+const char *host = "localhost";
+const char *port = "8000";
+uint8_t insecure = 1;
```

Comment:
This can be an option to turn off, but we need a way to do the full certificate verification since this will skip a non-trivial amount of work for the client. It looks like `argument_parse` can turn this on, but nothing ever turns it off. Since you have the certificate the server uses in the server code it is reasonable to have the public key of the root in the client to the client by default can verify the server.

Reply:
Currently working on getting secure mode to work! Current version will not support using secure mode.